### PR TITLE
Character extensions isNumber, isLetter, isLowercased, isUppercased a…

### PR DESCRIPTION
**Pull request by ShreeramBhat**
Removed character extensions that are added to stdlib.

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Hey @ShreeramBhat 
Thank's to taking the time, just two little things:
Can you add a changelog entry?
Can you run `swift test --generate-linuxmain` to update linux tests? 
👍

**Comment by ShreeramBhat** [comment]
Whenever I run the linux test script getting "dyld: Library not loaded: @rpath/libswiftCore.dylib
  Referenced from: /Library/Developer/CommandLineTools/usr/bin/swift-test
  Reason: image not found
Abort trap: 6"
Can you please tell me the exact steps to follow or point towards a resource. I am new to this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Deprecated
 
 ### Removed
+- **Character**:
+- **Breaking Change** Removed  `var isLowercased: -> Bool`, `var isUppercased: -> Bool` in favor of `var isLowercase: -> Bool`, `var isUppercase: -> Bool` in stdlib. [#675](https://github.com/SwifterSwift/SwifterSwift/pull/675) by [ShreeramBhat](https://github.com/ShreeramBhat)
+- Removed `var isLetter: -> Bool` ,  `var isWhiteSpace: -> Bool` in favor of `var isLetter: -> Bool`, `var isWhiteSpace: -> Bool` in stdlib. [#675](https://github.com/SwifterSwift/SwifterSwift/pull/675) by [ShreeramBhat](https://github.com/ShreeramBhat)
```
Comment (LucianoPAlmeida) [inline CHANGELOG.md]: Can you put these two on the same statement?

**Comment by LucianoPAlmeida** [comment]
🤔 
@ShreeramBhat It should be just run this command on the base path, seems like a toolchain issue. Which version of Xcode and command line tools are you using? 
Another easier way to solve this would just remove the lines referring to those tests removed on the Tests/SwiftStdlibTests/XCTestManifests.swift

File: Tests/SwiftStdlibTests/CharacterExtensionsTests.swift [hunk 2]
```diff
@@ -22,34 +22,7 @@ final class CharacterExtensionsTests: XCTestCase {
...
-    func testIsUpercased() {
-        XCTAssert(Character("S").isUppercased)
-        XCTAssertFalse(Character("s").isUppercased)
-    }
-
-    func testIsWhiteSpace() {
-        XCTAssertTrue(Character(" ").isWhiteSpace)
-        XCTAssertFalse(Character("-").isWhiteSpace)
-    }
+    }
```
Comment (guykogus) [inline Tests/SwiftStdlibTests/CharacterExtensionsTests.swift]: You have leading spaces here. There should be a SwiftLint warning about it.

**Comment by LucianoPAlmeida** [comment]
Seems like this became stale. Closing so other people can give a try on that one ;)
Feel free to reopen anytime if you like :))

**Pull request closed without merging**